uavcan/equipment/ecu/1120.Status.uavcan

```diff
@@ -0,0 +1,118 @@
+#
+# Generic status message of a piston engine control system.
+#
+
+#
+# General status flags.
+#
+uint8 FLAG_STARTING                     = 1
+uint8 FLAG_CRANKSHAFT_SENSOR_ERROR      = 2
+uint8 FLAG_OVERHEATING                  = 4
+uint8 flags
+
+#
+# Engine load estimate.
+# Units: percent.
+# Range: [0, 127].
+#
+uint7 engine_load_percent
+
+#
+# Engine speed.
+# Units: revolutions per minute.
+#
+uint17 engine_speed_rpm
+
+#
+# Spark dwell time.
+# Units: millisecond.
+#
+float16 spark_dwell_time_ms
+
+#
+# Atmospheric (barometric) pressure.
+# Units: kilopascal.
+#
+float16 atmospheric_pressure_kpa
+
+#
+# Engine intake manifold pressure.
+# Units: kilopascal.
+#
+float16 intake_manifold_pressure_kpa
+
+#
+# Engine intake manifold temperature.
+# Units: kelvin.
+#
+float16 intake_manifold_temperature
+
+#
+# Engine coolant temperature.
+# Units: kelvin.
+#
+float16 coolant_temperature
+
+#
+# Exhaust gas temperature (EGT).
+# Units: kelvin.
+#
+float16[<=16] exhaust_gas_temperature

```

Comment:
Not necessarily.
Number of exhaust gas temperatures is usually less or equal to the number of cylinders.
Depends on engine construction.